package.json

```diff
@@ -14,18 +14,18 @@
   "sideEffects": false,
   "scripts": {
     "prebuild": "rimraf ./dist",
-    "dev": "yarn dev:js & yarn dev:scss & yarn dev:css",
+    "dev": "concurrently -n js,css,scss 'yarn dev:js' 'yarn dev:css' 'yarn dev:scss'",
```

Comment:
Should we define `concurrently` as a specific callout on how to use and troubleshoot

Reply:
nope